+neuron/NrnRef.m

```diff
@@ -0,0 +1,82 @@
+classdef NrnRef < handle
+% Neuron reference (NrnRef) wrapper class. The referenced value can be
+% accessed by indexing, e.g. ```t = n.ref('t'); disp(t(1));```.
+
+    properties (SetAccess=protected, GetAccess=public)
+        obj             % C++ NrnRef object.
+    end
+    properties (Dependent)
+        length          % Data length.
+    end
+    
+    methods
+        function self = NrnRef(obj)
+        % Initialize NrnRef
+        %   NrnRef(obj) constructs a Matlab wrapper for C++ NrnRef obj
+            self.obj = obj;
+        end
+        function value = get(self, ind)
```

Comment:
The help text for the get and set methods do no show the object itself. The set method help is quite incorrect even

Reply:
Fixed the arguments description for `set`. Let's make a new issue to rethink the way we write method help texts.